### ISSUE TITLE
Lack of visibility on how Credits are being used

### ISSUE THREAD

**Issue by kdybicz** [serverless/serverless#12699]
### Issue description

I was testing Serverless v4 in a single endpoint-stage configuration and for some reason, three Credits were used over the period of 2-3 weeks of tests and troubleshooting. In the Serverless Dashboard, there is no breakdown or summary, or anything else that would show me where the credits were allocated and why there were three Credits used instead of one.

### Context

When migrating to Serverless v$ you need to pay to get a pack of Credits that can be used to deploy your services. In theory, one credit is used to deploy and maintain one instance of a service for a month. Sometimes, for some reason that's not the case, and there is no means of checking what happened and where those credits go.

**Comment by kdybicz** [comment]
This is a serious problem, as right now we don't have any service deployed using SLS v4 and yet you took 2 out of 15 credits we purchased, and we don't know how nor why. A while back we had only one service instance deployed in one environment for a while and you "charged" us 3 credits for that, so now being down to two is an improvement I guess...

**Comment by eahefnawy** [comment]
Hi @kdybicz, thanks for your feedback. We understand and apologise for your frustration. The dashboard currently only lists the instances using dashboard features, but we are working on a better usage view that would list all your instances, including v4 instances. Stay tuned 🙌

**Comment by kdybicz** [comment]
I believe this could already be closed? :)

**Comment by czubocha** [comment]
Yeah, we added a "Usage" tab in the dashboard where all instances are now listed, including those created via CLI. Closing this out :)
